jsonschema-core.xml

```diff
@@ -127,19 +127,36 @@
 
         <section title="Overview">
             <t>
-                This document proposes a new media type "application/schema+json" to identify a JSON
-                Schema for describing JSON data.
-                It also proposes a further optional media type, "application/schema-instance+json",
-                to provide additional integration features.
+                This document specifies the media type "application/schema+json" which
+                can describe a set of JSON documents, classify documents as instances of this set,
+                and to generate annotations about a given instance.
+                It also specifies "application/schema-instance+json",
+                to describe JSON documents known to be instances of a particular schema.
                 JSON Schemas are themselves JSON documents.
-                This, and related specifications, define keywords allowing authors to describe JSON
-                data in several ways.
             </t>
             <t>
-                JSON Schema uses keywords to assert constraints on JSON instances or annotate those
-                instances with additional information.  Additional keywords are used to apply
-                assertions and annotations to more complex JSON data structures, or based on
-                some sort of condition.
+                A JSON Schema document consists of a set of keywords,
+                typically encoded as properties in a JSON object.
+                The name of the keyword is used as the property name,
+                and any arguments to the keyword are provided as the value.
+            </t>
+            <t>
+                Each keyword provides an assertion and/or annotations about the instance.
+                Assertions add constraints that instances must conform to.
+                Given a schema and an instance, the schema "accepts" an instance whenever all the assertions are met,
+                and the schema "rejects" when any of the assertions fail.
+                Schemas may also be used to build a set of JSON documents:
```

Comment:
I think this should end with a `.` rather than a `:`.

```suggestion
                Schemas may also be used to build a set of JSON documents.
```